docs/user-journeys/users/application-developer/foundational.md

```diff
@@ -9,80 +9,151 @@ track: "USERS › APPLICATION DEVELOPER › FOUNDATIONAL"
+You can get basic information about your cluster with the command `kubectl get nodes`. To get a good idea of what's really going on, however, you need to deploy an application to your cluster. This is covered in the next major section.
+
+## Deploy an application
+
+#### A Basic Workload
+
+The following examples demonstrate how to deploy basic apps to Kubernetes:
+  * [Deploy a simple, stateless nginx server](/docs/tasks/run-application/run-stateless-application-deployment/){:target="_blank"}. Nginx is often used as the web server to help provide ingress from outside your cluster (in other words, external requests).
+
+  * [Deploy a stateful, externally accessible MySQL database](/docs/tasks/run-application/run-single-instance-stateful-application/){:target="_blank"}. This introduces the concept of storage.
+
+Through these deployment tasks, you'll gain familiarity with the following concepts:
+* Fundamentals
+
+  * **Kubernetes manifests** - These configuration files describe the desired state of your application and related resources as Kubernetes should maintain them. They are written in YAML or JSON. (See the [example from the nginx app](/docs/tasks/run-application/run-stateless-application-deployment/#creating-and-exploring-an-nginx-deployment))
+
+  * **{% glossary_tooltip text="Pods" term_id="pod" %}** - This is the basic unit for all of the workloads you run on Kubernetes. You may find this [explanation of Pods and Nodes](/docs/tutorials/kubernetes-basics/explore-intro/){:target="_blank"} helpful.
+
+* Common workload objects
+  * **{% glossary_tooltip text="Deployment" term_id="deployment" %}** - The most common way of running X copies of your application. Supports rolling updates to your container images.
 
-Labels let you specify identifying information for Kubernetes objects that you define, such as pods. This information is entirely user-defined, and is used to sort and select sets of objects.
+  * **{% glossary_tooltip text="Ingress" term_id="ingress" %}** - A general API object that determines routing rules, such as how to handle traffic requests between your cluster and the outside world.
+
+  * **{% glossary_tooltip text="Service" term_id="deployment" %}** - By itself, a Deployment can't provide ingress. This is one of the simplest ways to configure ingress, as it supports load balancing and {% glossary_tooltip text="label" term_id="labels" %}  selection.
+
+The subsequent sections here describe other useful areas to know for app deployment.
+
+#### Metadata
+
```

Comment:
This section seems to be about labels, but not about any other kind of metadata. Let's use the heading **Labels** instead of the heading **Metadata**.